406-fibo-finance

fkim7 opened the pull request.

BenjaminMoe reviewed (changes requested):
Looks like there are some errors with the CI. https://github.com/w3c-ccg/traceability-vocab/actions/runs/3593988961/jobs/6118985939

Edit: we can help with tooling if you have any questions.

TallTed commented on docs/openapi/components/schemas/common/BankAccount.yml:
```diff
@@ -0,0 +1,76 @@
+      '@id': 'https://w3id.org/bankAccount#accountId'
+  BIC11:
+    type: string
+    description: Business Identifier Code- ISO 9362 identification of the financial institution’s branch.
```
```suggestion
    description: Business Identifier Code — ISO 9362 identification of the financial institution’s branch.
```

TallTed commented on docs/openapi/components/schemas/common/BankAccount.yml:
```diff
@@ -0,0 +1,76 @@
+      '@id': 'https://w3id.org/bankAccount#BIC11'
+  iban:
+    type: string
+    description: International Bank Account Number- ISO 13616 identification of bank accounts across national borders.
```
```suggestion
    description: International Bank Account Number — ISO 13616 identification of bank accounts across national borders.
```

nissimsan commented:
@fkim7 , we're discussing this on the call. We are happy to help get the issue resolved. You are also very welcome to join the weekly call. 
COntent looks good - we'd love to merge this.

fkim7 commented:
Yes I saw the invalid IRI- Where can I put the content of https://knox-networks.github.io/schema/bankaccount/v1.jsonld ?

BenjaminMoe reviewed (approved):
Looks like this is from a fork. I'd be okay with merging this if we make an issue to address an CI issue after merge.

fkim7 commented:
> > Yes I saw the invalid IRI- Where can I put the content of https://knox-networks.github.io/schema/bankaccount/v1.jsonld ?
> 
> You can put the content of the `context` json, in here: https://github.com/w3c-ccg/traceability-vocab/blob/main/packages/traceability-schemas/services/contexts.js#L23

Thank you! Updated context to be under https://w3id.org/traceability/v1 and updated the file name under /credential

Can someone review and approve & merge if all looks good?

BenjaminMoe commented:
> Can someone review and approve & merge if all looks good?

@fkim7, we'll review it on the call in an about an hour which you're welcome to come if you want to attend: https://github.com/w3c-ccg/traceability-vocab#meetings

nissimsan commented:
Merging, @BenjaminMoe , please fix the CI issues subsequently
